src/DataFixtures/SponsorshipBenefitFixtures.php

```diff
@@ -22,11 +22,12 @@ public function __construct(SponsorshipBenefitManagerInterface $sponsorshipBenef
 
     /**
      * @param ObjectManager $manager
+     *
      * @throws \Exception
      */
-    public function load(ObjectManager $manager)
+    public function load(ObjectManager $manager): void
     {
-        for ($sb = 0; $sb < self::NB_SPONSORSHIP_BENEFIT; ++$sb) {
+        for ($sb = 0; $sb < self::NB_SPONSORSHIP_BENEFIT; $sb++) {
```

Comment:
Beware: changing from pre to post will change index from 1 to 0 basically. Also, `$sb` is a pretty bad name.

Reply:
Same as above about pre to post
I agree about $sb, i am going to fix it